<launch>
  <include file="$(find realsense2_camera)/launch/rs_camera.launch" pass_all_args="true"/>
  <arg name="filters" default="pointcloud"/>
  <arg name="model" default="$(find arm_control)/urdf/arm.urdf"/>
  <arg name="gui" default="true" />
  <arg name="rvizconfig" default="$(find arm_control)/rviz/arm_sensor.rviz"/>

  <param name="robot_description" command="$(find xacro)/xacro.py $(arg model)" />
  <!-- <param name="use_gui" value="$(arg gui)"/>
  <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" /> -->
  <node name="ArmRos" pkg="arm_control" type="ArmRos.py" />
  <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" />
  <node name="rviz" pkg="rviz" type="rviz" args="-d $(arg rvizconfig)" required="true" />

</launch>
